apps/macro-sandbox/functions/python/handler.py

```
@@ -0,0 +1,126 @@
+
+
+def _execute(event):
+    _cleanup_stale_tmp()
+
+    # Validate script
+    if "script" not in event:
+        return {"status": "error", "results": [], "errors": ["Missing 'script' field"]}
+
+    try:
+        script_bytes = base64.b64decode(event["script"])
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)